projects/1.19/assets/croptopia/croptopia/lang/zh_cn.json

```
@@ -0,0 +1,631 @@
+  "item.croptopia.peanut": "花生",
+  "item.croptopia.pineapple": "菠萝",
+  "item.croptopia.radish": "小萝卜",
+  "item.croptopia.raspberry": "树莓",
+  "item.croptopia.rhubarb": "大黄",
+  "item.croptopia.rice": "大米",
+  "item.croptopia.rutabaga": "芜菁甘蓝",
+  "item.croptopia.saguaro": "巨人柱",
+  "item.croptopia.spinach": "菠菜",
+  "item.croptopia.squash": "倭瓜",
+  "item.croptopia.strawberry": "草莓",
+  "item.croptopia.sweetpotato": "红薯",
+  "item.croptopia.tomatillo": "酸浆果",
+  "item.croptopia.tomato": "番茄",
+  "item.croptopia.turnip": "芜菁",
+  "item.croptopia.yam": "山药",
+  "item.croptopia.zucchini": "西葫芦",
+  "item.croptopia.artichoke_seed": "洋蓟种子",
+  "item.croptopia.asparagus_seed": "芦笋种子",
+  "item.croptopia.bellpepper_seed": "甜椒种子",
+  "item.croptopia.blackbean_seed": "黑豆种子",
+  "item.croptopia.blackberry_seed": "黑莓种子",
+  "item.croptopia.blueberry_seed": "蓝莓种子",
+  "item.croptopia.broccoli_seed": "西兰花种子",
+  "item.croptopia.cabbage_seed": "卷心菜种子",
+  "item.croptopia.cantaloupe_seed": "哈密瓜种子",
+  "item.croptopia.cauliflower_seed": "花椰菜种子",
+  "item.croptopia.celery_seed": "芹菜种子",
+  "item.croptopia.coffee_seed": "咖啡种子",
+  "item.croptopia.corn_seed": "玉米种子",
+  "item.croptopia.cranberry_seed": "蔓越莓种子",
+  "item.croptopia.cucumber_seed": "黄瓜种子",
+  "item.croptopia.currant_seed": "醋栗种子",
+  "item.croptopia.eggplant_seed": "茄子种子",
+  "item.croptopia.elderberry_seed": "接骨木莓种子",
+  "item.croptopia.garlic_seed": "大蒜种子",
+  "item.croptopia.grape_seed": "葡萄种子",
+  "item.croptopia.greenbean_seed": "绿豆种子",
+  "item.croptopia.greenonion_seed": "大葱种子",
+  "item.croptopia.honeydew_seed": "蜜瓜种子",
+  "item.croptopia.hops_seed": "啤酒花种子",
+  "item.croptopia.kale_seed": "羽衣甘蓝种子",
+  "item.croptopia.kiwi_seed": "猕猴桃种子",
+  "item.croptopia.leek_seed": "韭葱种子",
+  "item.croptopia.lettuce_seed": "生菜种子",
+  "item.croptopia.olive_seed": "橄榄种子",
+  "item.croptopia.onion_seed": "洋葱种子",
+  "item.croptopia.peanut_seed": "花生种子",
+  "item.croptopia.pineapple_seed": "菠萝种子",
+  "item.croptopia.radish_seed": "小萝卜种子",
+  "item.croptopia.raspberry_seed": "树莓种子",
+  "item.croptopia.rhubarb_seed": "大黄种子",
+  "item.croptopia.rice_seed": "大米种子",
+  "item.croptopia.rutabaga_seed": "芜菁甘蓝种子",
+  "item.croptopia.saguaro_seed": "巨人柱种子",
+  "item.croptopia.spinach_seed": "菠菜种子",
+  "item.croptopia.squash_seed": "倭瓜种子",
+  "item.croptopia.strawberry_seed": "草莓种子",
+  "item.croptopia.sweetpotato_seed": "红薯种子",
+  "item.croptopia.tomatillo_seed": "酸浆果种子",
+  "item.croptopia.tomato_seed": "番茄种子",
+  "item.croptopia.turnip_seed": "芜菁种子",
+  "item.croptopia.yam_seed": "山药种子",
+  "item.croptopia.zucchini_seed": "西葫芦种子",
+  "item.croptopia.oat_seed": "燕麦种子",
+  "item.croptopia.mustard_seed": "芥菜种子",
+  "item.croptopia.pepper_seed": "胡椒种子",
+  "item.croptopia.turmeric_seed": "姜黄种子",
+  "item.croptopia.ginger_seed": "姜种子",
+  "item.croptopia.basil_seed": "罗勒种子",
+  "item.croptopia.chile_pepper_seed": "辣椒种子",
+  "item.croptopia.barley_seed": "大麦种子",
+  "item.croptopia.soybean_seed": "黄豆种子",

```

```suggestion
  "item.croptopia.soybean_seed": "大豆种子",
```
[大豆（百度百科）](https://baike.baidu.com/link?url=BkQcq968RqEYSk0GpVL3wIp46xE9IMF_Hi9pVXnCMn2oSlPszMgSH8lP9J4OLFCSVubJqSoe_Z58qmqC_4JPTSaN0iqLZRUPTLVWqc5Hz3K)